api/core/model_runtime/model_providers/openai_api_compatible/text_embedding/text_embedding.py

```diff
@@ -161,7 +161,7 @@ def validate_credentials(self, model: str, credentials: dict) -> None:
             endpoint_url = urljoin(endpoint_url, 'embeddings')
 
             payload = {
-                'input': 'ping',
+                'input': ['ping'],
```

Comment:
According to this doc https://platform.openai.com/docs/api-reference/embeddings/create#embeddings-create-input
It should be a string or an array.